src/config/Config.zig

```diff
@@ -1886,6 +1886,20 @@ keybind: Keybinds = .{},
     else => .false,
 },
 
+/// The action to take when the user right-clicks on the terminal surface.
+///
+/// Valid values:
+///   * `context-menu` - Show the context menu.
+///   * `paste` - Paste the contents of the clipboard.
+///   * `copy` - Copy the selected text to the clipboard.
+///   * `copy-and-paste` - Depending on the current selection:
+///         * Some text selected: Copy the selected text to the clipboard.
+///         * No selection: Paste the contents of the clipboard.
```

Comment:
This is more accurately copy ***or*** paste

```suggestion
///   * `copy-or-paste` - If there is a selection, copy the selected text to the clipboard; otherwise, paste the contents of the clipboard.
```